test/fixtures/amp-bind-integrations.html

```diff
@@ -136,5 +134,11 @@
   <amp-list id='list' src="https://www.google.com/unbound.json" [src]=listSrc>
   </amp-list>
 
+  <p>AMP-STATE</p>
+  <button id="ampStateSrcButton" on="tap:AMP.setState({ampState: {ampStateSrc: 'https://www.google.com/bind/second/source'}})">Change amp-state</button>
```

Comment:
that would be inconsistent with the rest of the ids int he file that describe the element that will have its `src` changed.

'stateSrcButton'?